api/api/v1alpha2/astarte_types.go

```diff
@@ -1,19 +1,19 @@
 /*
-  This file is part of Astarte.
+This file is part of Astarte.
 
-  Copyright 2020-23 SECO Mind Srl
+Copyright 2024 SECO Mind Srl.
```

Comment:
Nit: should we keep the old copyright starting year if the file was moved?

Reply:
Also
```suggestion
Copyright 2025 SECO Mind Srl
```

Reply:
You're right folks! Working on a solution.

Reply:
After an IRL disussion, I changed this line to `Copyright 2020-25 SECO Mind Srl`.